### PR TITLE
[EDU-1613] Add section for pricing

### PR DESCRIPTION
**Pull request by m-hulbert**
## Description

This PR adds a section on Pricing with the following content:

* Overview
* Free package
* Standard package
* Pro package
* Enterprise package
* Billing and invoicing
* Limits (updated)
* Pricing FAQs

### CODE REVIEW

File: content/pricing/index.textile [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+There is also a charge on additional bandwidth consumed.
+
+This is worked out based on the average message size of 5KiB multiplied by the number of messages consumed in the month. If your total bandwidth for the month is less than this value then there is no charge for bandwidth. Anything over this value is charged at $0.25 per GiB.
+
+h2(#packages). Packages
+
+Ably offers four different package types so that you can choose the one that best suits your needs:
+
+* "Free":/pricing/free: for experimenting with Ably, or building a proof of concept. No credit card required.
```
Comment (GregHolmes) [inline content/pricing/index.textile:38]: Each of these bullet points have broken links. The trailing : at the end is considered part of the link, so it'll 404. You'll need to add a space between the url and the :

**Review by GregHolmes — DISMISSED** [review]

File: content/pricing/billing.textile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+For package upgrades, the base package price will be charged pro-rata from the point in the month that you upgraded. This is true for both base package prices if you are upgrading from a paid package.
+
+Package downgrades take effect on the 1st of the following month.
+
+<aside data-type='note'>
+<p>If you have an "Enterprise package":/pricing/enterprise with Ably, then contact your Customer Success Manager to discuss any details or queries you may have regarding your package.</p>
+</aside>
+
+h2(#billing). Billing details and invoices
```
Comment (kavalerov) [inline content/pricing/billing.textile:20]: Probably not in this but in a follow up PR we should add note re organizations (that billing is configured for each account separately)

File: content/pricing/faqs.textile [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+h3(#concurrent-connections). How do you count concurrent connections?
+
+The "limit":/pricing/limits#connection on concurrent connections is for the maximum number of realtime clients "connected":/connect to Ably simultaneously at any point in time. HTTP requests such as those from REST clients do not count towards this number, as it is solely related to realtime connections.
+
+h3(#concurrent-channels). How do you count concurrent channels?
+
+The "limit":/pricing/limits#channel on concurrent channels is for the maximum number of channels that are active simultaneously at any point in time.
+
+"Channels":/channels are opened when either a messaged is published on the channel using a REST client, or a realtime client attaches to the channel. They are considered active until the channel is closed.
```
Comment (kavalerov) [inline content/pricing/faqs.textile:54]:
```suggestion
"Channels":/channels are opened when either a message is published on the channel using a REST client, or a realtime client attaches to the channel. They are considered active until the channel is closed.
```

**Review by kavalerov — CHANGES_REQUESTED** [review]

File: content/pricing/limits.textile [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+h2(#channel). Channel limits
+
+Channel limits relate to the number, rate and membership of "channels":/channels on your account.
+
+|_. Channel limit |_. Free |_. Standard |_. Pro |_. Enterprise |
+| *Concurrent channels*<p>_the maximum number of channels that are active simultaneously at any point_</p> | 200 | 10,000 | 50,000 | Unlimited |
+| *Default last message on channel TTL (days)*<p>_the default period that the last message on a channel is stored for, when enabled_</p> | 1 | 30 | 365 | 365 |
+| *Channel creation rate (per second)*<p>_the maximum rate at which channels can be created across an account_</p> | 42 | 250 | 500 | Custom |
+| *Message publish rate per channel (per second)*<p>_the maximum rate at which messages can be published for each channel_</p> | 50 | 50 | 50 | 50 |
+| *Presence members per channel*<p>_the maximum number of clients that can be simultaneously present on a channel_</p> | 200 | 200 | 200 | 200 |
```
[the remark this answers is not in the document]
Comment (kavalerov) [reply content/pricing/limits.textile:83]: Just looking at how it is written in the spreadsheet.

**Review by kavalerov — APPROVED** [review]

**Pull request merged**